code/game/objects/items/stacks/medical.dm

```diff
@@ -249,12 +249,13 @@
 	singular_name = "Soteria medical gauze"
 	desc = "An advanced sterile gauze to wrap around bloody stumps. Unlike the regular gauze, these have more charges, and sterilize wounds as ointment would, as well as treating small amounts of toxins. Hand-made, with love, by Soteria Medical staff."
 	icon_state = "sr_brutepack"
+	preloaded_reagents = list("quickclot" = 5, "sterilizine" = 10)
```

Comment:
these dont inject on healing, meaning grinding them up gives these chems otherwise they have no use

Reply:
Aware, they're there for fluff reasons, as most medical gauzes irl ARE dosed with quicklot, and the sterilizine is from the crystals